ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2460,5 +2461,94 @@
       "version": "3.3.9"
     }
   ]
-}
+  },
+  {
+  "id": "CentOS nodejs",
```

Comment:
All stacks, except centos nodejs and wildfly swarm have lowercase IDs without spaces. Can you please change the IDs into:
- nodejs4
- wildfly-swarm